app/client/cypress/integration/home/links.spec.js

```diff
@@ -17,4 +17,28 @@ describe('Homepage', () => {
     cy.findByText('National').click();
     cy.url().should('include', '/national');
   });
+  it(`"Data" button links to the data page`, () => {
+    cy.findByText('Data').click();
+    cy.url().should('include', '/data');
+  });
+  it(`"About" button links to the about page`, () => {
+    cy.findByText('About').click();
+    cy.url().should('include', '/about');
+  });
+  it(`"Contact Us" button links to the contact us page in a new window`, () => {
+    // Cypress does not have multi-tab support, verify the href and target, don't click
+    cy.findByText('Contact Us')
+      .should('have.attr', 'href')
+      .and(
+        'equal',
+        'https://www.epa.gov/waterdata/forms/contact-us-about-hows-my-waterway',
+      );
+    cy.findByText('Contact Us')
+      .should('have.attr', 'target')
+      .and('equal', '_blank');
+  });
+  it(`"How’s My Waterway?" header text links to home page`, () => {
```

Comment:
Add blank line between tests.
```suggestion
  });
  
  it(`"How’s My Waterway?" header text links to home page`, () => {
```